libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/VanishingRouteLine.kt

```diff
@@ -118,7 +118,7 @@ internal class VanishingRouteLine {
         ) { granularDistances, index ->
             val upcomingIndex = granularDistances.distancesArray[index]
             if (upcomingIndex == null) {
-                LoggerProvider.logger.e(
+                LoggerProvider.logger.d(
```

Comment:
Changing the log level will not resolve the ANR caused by the underlying string cosntruction.

We need to refactor the way we create the log message to thin it down a little bit.

Reply:
This isn't a logging message I've ever needed to use and it doesn't appear useful for a developer.  I"m removing the array values from the logging message.  It's easy enough to put them back while debugging an issue if needed. If there's a customer issue we are usually successful at getting the route and/or history and can recreate the necessary conditions. If you have other ideas let me know.